local-source-proxy/install.hbs.md

```diff
@@ -28,7 +28,10 @@ local_source_proxy:
 Where:
 
 - `local_source_proxy.repository` is required. This is the repository where all your source code
-  will be uploaded.
+  will be uploaded. Examples:
+  - Harbor has the form `my-harbor.io/my-project/local-source`.
+  - Docker Hub has the form `my-dockerhub-user/local-source` or `index.docker.io/my-user/local-source`.
+  - Google Cloud Registry has the form `gcr.io/my-project/local-source`.
```

Comment:
Lets add Google Artifact Registry here as well.

- Google Artifact Registry has the form `region-docker.pkg.dev/my-project/local-source/image`